testing/cockroach/ldflags_fix.patch

```diff
@@ -0,0 +1,18 @@
+--- a/Makefile
++++ b/Makefile
+@@ -78,12 +78,12 @@
+ build: install
+ 
+ .PHONY: install
+-install: LDFLAGS += $(shell GOPATH=${GOPATH} build/ldflags.sh)
++install: GOLDFLAGS := $(shell GOPATH=${GOPATH} build/ldflags.sh)
```

Comment:
This has been fixed upstream, FWIW. https://github.com/cockroachdb/cockroach/pull/14257